exercises/practice/word-count/.docs/instructions.md

```diff
@@ -13,9 +13,10 @@ When counting words you can assume the following rules:
 1. The count is _case insensitive_ (ie "You", "you", and "YOU" are 3 uses of the same word)
 2. The count is _unordered_; the tests will ignore how words and counts are ordered
 3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are ignored
-4. The words can be separated by _any_ form of whitespace (ie "\t", "\n", " ")
+4. The words can be separated by _any_ form of whitespace (i.e. "\t", "\n", " "), or
+   punctutaion (except for single apostrophe in the contraction which is part of the word).
```

Comment:
```suggestion
   external punctuation.
```

- fixed typo 
- phrasing: "external punctuation" seems clearer than "any punctuation except for single apostrophe". It's also more correct: given the input "He attended the barn-raising.", I'd expect the last word to be parsed as a single word "barn-raising", not as the two words "barn" and "raising".

Reply:
Updated.